modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Options/Options.es.js

```diff
@@ -340,7 +331,7 @@ const Main = ({
 								defaultOptionRef.current = false;
 							}
 						}}
-						onKeywordBlur={handleBlur}
+						onKeywordBlur={() => {}}
```

Comment:
@carolmariaabb, if there are no more handlers, what it's the purpose of keeping the prop `onKeywordBlur`?